### PR TITLE
Add support for High Sierra 10.13 and update README

### LINKED ISSUE #28: High Sierra 10.13
It seems that macOS 10.13 has changed the installer and BaseSystem.dmg is no longer in the InstallESD.dmg. I tried to make an iso using the BaseSystem.dmg/.chunklist that is inside the installer, but when the installation starts (after the selection of the language) an error appears that OSXInstall.mpkg is broken.

Any idea how to create macOS 10.13 beta iso from the installation app?

### PR DESCRIPTION
**Pull request by dotCipher**
Based on #37 and from some of the things outlined in #28.

Additions include new section in troubleshooting and fixed formatting (tabs to spaces) on the `prepare-iso.sh` script.

Fixes #28.

### CODE REVIEW

**Comment by eelkeblok** [comment]
I ran into a small issue with the auto-discovery of the installer. I fixed the issue in my own fork: https://github.com/eelkeblok/macos-virtualbox-vm/tree/highsierra-10.13-and-docs

(Not sure what the best way is to collaborate on a pull request; I think neither is Github 😄)

**Comment by dotCipher** [comment]
@eelkeblok I just pulled your forked branch commits in, thanks!

**Comment by dack** [comment]
@dotCipher - Just want to confirm, have you tested an actual install of the guest? The installer is not working for me on VirtualBox 5.1.30 (it reboot loops before launching the installer GUI). If it's known working for you, I'll have to test on an earlier VirtualBox version and/or a different host.

**Comment by dotCipher** [comment]
@dack I referenced in https://github.com/geerlingguy/macos-virtualbox-vm/issues/28 that actually I ran into problems when trying to do the install. I am using VirtualBox version: `5.0.20 r106931` on Mac OS X `10.12.16`

**Comment by eelkeblok** [comment]
I've been able to do a successful install. I did run into the issue of looping back into the install, but the readme suggested to remove the install media from the virtual box in the troubleshooting section, which did the trick for me.

**Comment by dotCipher** [comment]
Thanks @saitooo, I merged it.  Feedback from @geerlingguy would be appreciated here :)

File: prepare-iso.sh [hunk 1]
```diff
@@ -100,10 +118,10 @@ function createISO()
     hdiutil resize -size `hdiutil resize -limits /tmp/${isoName}.sparseimage | tail -n 1 | awk '{ print $1 }'`b /tmp/${isoName}.sparseimage
 
     echo
-    echo Convert the sparse bundle to ISO/CD master
+    echo Convert the ${isoName} sparse bundle to ISO/CD master
     echo --------------------------------------------------------------------------
-    echo $ hdiutil convert /tmp/${isoName}.sparseimage -format UDTO -o /tmp/${isoName}
-    hdiutil convert /tmp/${isoName}.sparseimage -format UDTO -o /tmp/${isoName}
+    echo $ time hdiutil convert /tmp/${isoName}.sparseimage -format UDTO -o /tmp/${isoName}
```
Comment (geerlingguy) [inline prepare-iso.sh]: Do these need to be in here? Looks like they were added for debugging purposes.

**Comment by geerlingguy** [comment]
Gah, there is also a merge conflict now.

**Comment by geerlingguy** [comment]
So... I'm getting the following message repeated ad infinitum when I first boot the VM with the High Sierra ISO mounted:

```
Attempting system restart...MACH Reboot
```

**Comment by geerlingguy** [comment]
Moving work to #51.

**Pull request closed without merging**